src/components/stats/PieChart.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Chart } from 'react-google-charts';
+
+interface PieChartProps {
+  data: { label: string; value: number }[];
+  title?: string;
+  width?: number;
+  height?: number;
+}
+
+const PieChart: React.FC<PieChartProps> = ({
+  data,
+  title = 'Pie Chart',
+  width = 600,
+  height = 400,
+}) => {
+  // Format the data for Google Charts
+  const chartData = [
+    ['Label', 'Value'],
+    ...data.map(item => [item.label, item.value]),
+  ];
+
+  const options = {
```

Comment:
nit: not a concern, but is there an easy way to change the font to Poppins? No worries if not